src/chain/chain_accumulator.rs

```diff
@@ -30,6 +67,14 @@ impl ChainAccumulator {
         event: AccumulatingEvent,
         proof_set: ProofSet,
     ) -> Result<(), InsertError> {
+        // The entry must be pre-existed. Otherwise just don't track.
+        let voters: Vec<_> = proof_set.ids().cloned().collect();
+
+        let _ = self
+            .vote_statuses
+            .iter_mut()
+            .any(|vote_status| vote_status.add_votes(&event, &voters));
```

Comment:
Why is this not only for completed votes?
Also why do this, we do not call it again for supermajority consensus item (Add/Remove)